tests/test_downloader.py

```
@@ -5012,3 +5105,1152 @@ def test_get_release_tuple_whitespace_handling(self):
+            with patch("requests.Session") as mock_session_class:
+                mock_session = Mock()
+                mock_session_class.return_value = mock_session
+
+                mock_response = Mock()
+                mock_response.status_code = 200
+                mock_response.headers = {"content-length": "1024"}
+
+                # Simulate interruption after some chunks
+                def mock_iter_content(_chunk_size):
+                    yield b"data" * 10
+                    raise requests.exceptions.ConnectionError  # noqa: TRY003
```

Ruff (0.14.1)

Unused `noqa` directive (unused: `TRY003`)

Remove unused `noqa` directive

(RUF100)